stingray/crossspectrum.py

```diff
@@ -225,6 +254,27 @@ def _normalize_crossspectrum(self, unnorm_cs, tseg):
 
         return cs
 
+    def coherence(self):
+        """
+        Compute Coherence function of the cross spectrum. Coherence is a
+        Fourier frequency dependent measure of the linear correlation
+        between time series measured simultaneously in two energy channels.
+
+        Returns
+        -------
+        coh : numpy.ndarray
+            Coherence function
+
+        References
+        ----------
+        .. [1] http://iopscience.iop.org/article/10.1086/310430/pdf
+
+        """
+        ps1 = Powerspectrum(self.lc1)
+        ps2 = Powerspectrum(self.lc2)
+
+        return self.cs/(ps1.ps * ps2.ps)
```

Comment:
But currently, we don't have the `norm='none'` option available in `Powerspectrum`. We can use `ps.unnorm_powers` instead.